GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1109,7 +1126,7 @@ int SearchRevision(ObjectId objectId)
 
         [ContractAnnotation("=>false,parentIds:null")]
         [ContractAnnotation("=>true,parentIds:notnull")]
```

Comment:
These annotations are no longer correct. Should be just `[NotNull]`.

Reply:
Fixed directly in master